apps/expo-go/android/expoview/src/main/java/host/exp/exponent/exceptions/ExceptionUtils.kt

```diff
@@ -61,13 +61,28 @@ object ExceptionUtils {
     return null
   }
 
+  fun exceptionToPlainText(exception: Exception): String {
+    if (exception is ManifestException) {
+      return "${exceptionToErrorHeader(exception)}\n\n${replaceHtml(exception.errorMessage)
+      }\n\nHow to fix this error:\n${replaceHtml(exception.fixInstructions)}"
+    }
+    return exception.toString()
+  }
+
   fun exceptionToCanRetry(exception: Exception): Boolean {
     if (exception is ManifestException) {
       return exception.canRetry
     }
     return true
   }
 
+  private fun replaceHtml(string: String?): String {
+    return string
+      ?.replace("<br>", "\n")
+      ?.replace("<b>", "")
+      ?.replace("</b>", "") ?: ""
+  }
```

Comment:
not sure whether [Html](https://developer.android.com/reference/android/text/Html.html) or [HtmlCompat](https://developer.android.com/reference/androidx/core/text/HtmlCompat) would help to strip html tags

Reply:
I suggest that we merge and get this PR out and then I'll make another one refactoring `ManifestException` on both Android/iOS (as suggested in #36170)